src/bootlint.js

```diff
@@ -1188,6 +1188,9 @@ var LocationIndex = _location.LocationIndex;
                         seenLint = true;
                     }
 
+                    // Append the wiki URL to the lint message.
```

Comment:
I would prefer setting a `.url` property in the `LintError` and `LintWarning` constructors so it's available to other reporters